.travis.yml

```diff
@@ -63,10 +63,10 @@ jobs:
       env: TARGETS="-C libbeat testsuite"
       go: $GO_VERSION
       stage: test
-    - os: linux
-      env: TARGETS="-C libbeat crosscompile"
-      go: $GO_VERSION
-      stage: test
+      #    - os: linux
```

Comment:
Maybe leave a comment here that links to https://github.com/elastic/beats/issues/5984